src/js/view/view.js

```diff
@@ -868,14 +868,21 @@ function View(_api, _model) {
             _this.trigger(FLOAT, { floating: true });
             _model.set('floating', true);
 
-            _resizeOnFloat = true;
+            if (_isMobile) {
+                addClass(_wrapperElement, 'jw-float-top');
+            } else {
+                addClass(_wrapperElement, 'jw-float-bottom');
+                _resizeOnFloat = true;
 
-            // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
-            const { width, height } = _this.getSafeRegion(false);
-            const ratio = Math.min(1, MAX_FLOATING_WIDTH / width, MAX_FLOATING_HEIGHT / height);
-            _this.resize(width * ratio, height * ratio, true);
+                // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
+                const { width, height } = _this.getSafeRegion(false);
+                const ratio = Math.min(1, MAX_FLOATING_WIDTH / width, MAX_FLOATING_HEIGHT / height);
```

Comment:
@DanFerrer Shouldn't the ratio be width/height ?  We would want the the proportion of the height to the width to be constant, right ?

Reply:
All you need is this:
```js
const floatingWidth = Math.min(_lastWidth, MAX_FLOATING_WIDTH);
const floatingHeight = Math.min(_lastHeight * floatingWidth / _lastWidth, MAX_FLOATING_HEIGHT);
_this.resize(floatingWidth, floatingHeight, true);
```

Reply:
If you actually care about which of x and y ratio, then you have to apply the x ratio to height when x ratio is smaller, or y ratio to width when it is smaller.